### PR TITLE
Test Prow and Fix Revealed Errors

### PR DESCRIPTION
**Pull request by brian-avery**
This adds support for the vale linter and fixes errors in spelling and links discovered through it. This linter can be expanded upon to automate some document verification.

### CODE REVIEW

File: topics/docs/monitoring_and_tracing/grafana.adoc [hunk 1]
```diff
@@ -20,12 +20,10 @@ A route to access Grafana UI should already exist. Query OpenShift for details o
 
 ```
 
-Open `http://grafana-istio-system.127.0.0.1.nip.io` in a browser. You should see Grafana's home screen, similar to this:
+Open `\http://grafana-istio-system.127.0.0.1.nip.io` in a browser. You should see Grafana's home screen, similar to this:
```
Comment (dmitri-d) [inline topics/docs/monitoring_and_tracing/grafana.adoc:23]: An errant backslash?

Comment (dmitri-d) [reply topics/docs/monitoring_and_tracing/grafana.adoc:23]: nm!

**Review by dmitri-d — APPROVED** [review]

File: vocab.txt [hunk 2]
```diff
@@ -13,3 +16,6 @@ Maistra
 multitenant
 Multitenancy
 nftables
+productpage
+unencrypted
+schedulable
```
Comment (dgn) [inline vocab.txt:21]: should we add `OpenShift Service Mesh` here?

**Review by dgn — APPROVED** [review]
lgtm, just one minor comment

File: vocab.txt (hunk 2 above)
Comment (brian-avery) [reply vocab.txt:21]: We have no spelling errors for it :) We can add it in the future if we need to.

Comment (brian-avery) [reply vocab.txt:21]: This vocab.txt doesn't accept spaces btw.

Comment (dgn) [reply vocab.txt:21]: > This vocab.txt doesn't accept spaces btw.

ah I see

**Comment by brian-avery** [comment]
@mergifyio backport maistra-1.1

**Comment by brian-avery** [comment]
@mergifyio refresh

**Pull request merged**